src/api/api.ts

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios';
+const baseURL = 'http://localhost:4000';
+
+export const getProducts = () => {
+  return axios.get(`${baseURL}/products`, {

```

Comment:
better to extract initialization of axioms into api.ts and declare routes according to their domain, i.e. 

/api/api.ts
/api/products.ts

Reply:
Fixed